packages/hint-button-type/src/meta.ts

```diff
@@ -0,0 +1,15 @@
+import { Category } from 'hint/dist/src/lib/enums/category';
+import { HintScope } from 'hint/dist/src/lib/enums/hintscope';
+import { HintMetadata } from 'hint/dist/src/lib/types';
+
+const meta: HintMetadata = {
+    docs: {
+        category: Category.interoperability,
```

Comment:
What category should we use for this? Maybe it's time to create something new @webhintio/core ?

Reply:
Maybe. Not an easy one to name, though. I see the common thread as behavior that is consistent cross-browser, but unexpected. Perhaps “pitfalls”?

Reply:
@molant Unless you really want to create a new category I vote we leave this in `interoperability` for now. We can always recategorize it later if we get a few more hints this makes sense to group with.

Reply:
I think you created a couple more hints like this one but yes, this should be done in another PR. 
I've created [this issue](https://github.com/webhintio/hint/issues/1683) to track it.